jenkins-pipelines/build-dex.Jenkinsfile

```diff
@@ -0,0 +1,84 @@
+import java.text.SimpleDateFormat
+
+class Globals {
+    static String dockerTag = null
+    static String dockerCacheArg = null
+}
+
+def legion
+
+pipeline {
+    agent any
+    options{
+        buildDiscarder(logRotator(numToKeepStr: '5'))
+        disableConcurrentBuilds()
+    }
+    environment {
+        build_workspace = "${WORKSPACE}"
+        def dex_dockerimage = "k8s-dex"
```

Comment:
why do we need 'def' for dex_dockerimge?

Reply:
In this section not needed. Removed.